containers/mkv.rs

```diff
@@ -59,10 +63,10 @@ extern "C" fn read_callback(pos: c_longlong,
 
     unsafe {
         let reader: &mut Box<Box<StreamReader>> = mem::transmute(&mut user_data);
-        if reader.seek(pos, SeekStyle::SeekSet).is_err() {
+        if reader.seek(SeekFrom::Start(pos as u64)).is_err() {
             return -1
         }
-        match reader.read_at_least(len as usize, slice::from_raw_parts_mut(buf, len as usize)) {
+        match reader.read(slice::from_raw_parts_mut(buf, len as usize)) {
```

Comment:
Wait, I don't think this is right. Need to loop and keep asking for bytes until it's full. Seems to be working in my testing but probably super fragile.

Do you think we should have a little utils module to have this function?